### PR TITLE
Mangrove gmw

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

File: deafrica/data/gmw.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    s3_dump(
+        json.dumps(item.to_dict(), indent=2),
+        item.self_href,
+        ContentType="application/json",
+        ACL="bucket-owner-full-control",
+    )
+    log.info(f"STAC written to {item.self_href}")
+
+
+def download_gmw(year: str, s3_dst: str, crs="EPSG:6933", res=10):
```
Comment (NikitaGandhi) [inline deafrica/data/gmw.py]: this function does more than just `download_gmw` so please think better naming

File: deafrica/data/gmw.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        with open(local_filename, "wb") as f:
+            shutil.copyfileobj(r.raw, f)
+
+    with ZipFile(local_filename) as z:
+        z.extractall()
+
+    return [f for f in z.namelist() if f.endswith(".shp")][0]
+
+
+def create_item_and_dump(cog_file: str, s3_dst: str, year) -> Item:
```
Comment (NikitaGandhi) [inline deafrica/data/gmw.py]: come up with better function name... like `create_and_upload_stac` - pick a name that best describes the purpose

File: deafrica/data/gmw.py (hunk 1 above)
Comment (carioca-au) [reply deafrica/data/gmw.py]: Done

File: deafrica/data/gmw.py (hunk 2 above)
Comment (carioca-au) [reply deafrica/data/gmw.py]: DONE

**Review by NikitaGandhi — APPROVED** [review]

**Pull request merged**